consensus/consensus-types/src/common.rs

```diff
@@ -64,61 +78,79 @@ impl Payload {
                 .with_min_len(100)
                 .map(|txn| txn.raw_txn_bytes_len())
                 .sum(),
+            Payload::InQuorumStore(_) => 0, // quorum store TODO
```

Comment:
Ideas on how to resolve this TODO?

Reply:
Easy, we carry now the size information in the proofs. Fixed.